piqueserver/game_modes/freeforall.py

```diff
@@ -1,19 +1,41 @@
 """
 Free for All: shoot anyone
+
+Options
+^^^^^^^
+.. code-block:: toml
+    [freeforall]
+    # If ALWAYS_ENABLED is False, free for all can still be enabled in the map
+    # metadata by setting the key 'free_for_all' to True in the extensions
+    # dictionary
+    always_enabled = true
+
+    # If WATER_SPAWNS is True, then players can spawn in water
+    water_spawns = false
+
+    # If ISOLATE_PLAYER is True, then player will be alone in
+    # the blue team, vs all greens auto disabling use_score_hack
+    isolate_player = true
+
+    # If USE_SCORE_HACK is True, then when you kill someone
+    # he will get changed to the other team, so you can
```

Comment:
Avoid using "he" pronouns for the user, e.g.
```suggestion
    # that player will appear to be on the other team, so you can
```